ui/src/main/resources/PhenomeCentral/RemoteMatching.xml

```diff
@@ -483,15 +483,26 @@ td input[type="text"]:focus {
           #if ($serverName == "")
             #set ($serverName = $serverID)
           #end
-          #set ($resultsURL = $xwiki.getURL('PhenomeCentral.RemoteResults', 'get', ""))
```

Comment:
I think after the change [RemoteResults.xml](https://github.com/phenotips/remote-matching/blob/master/ui/src/main/resources/PhenomeCentral/RemoteResults.xml) is never used can can be removed?

Reply:
Also, I think [RemoteMatchingScriptService.java](https://github.com/phenotips/remote-matching/blob/master/core/client/src/main/java/org/phenotips/remote/client/script/RemoteMatchingScriptService.java) was only used by `RemoteResults.xml` and can also be removed (i.e. this script service was replaced by REST, which is good)